### ISSUE TITLE
Downloading tracks without names

### ISSUE THREAD

**Issue by fominator** [vitiko98/qobuz-dl#193]
After upgrade downloading tracks are displayed without names - 00.tmp 01.tmp 02.tmp etc
Is it normal?

**Comment by Click1701** [comment]
Yes, I also noticed that. You will then have the correct filename in the folder after the download is done.
